core/store/migrate/plugins/relayer/evm/testutils/db.go

```diff
@@ -0,0 +1,98 @@
+package testutils
+
+import (
+	"database/sql"
+	_ "embed"
+	"fmt"
+	"net/url"
+	"sync"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/scylladb/go-reflectx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/core/config/env"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/v2/core/store/dialects"
+	"github.com/smartcontractkit/chainlink/v2/core/store/migrate/plugins/relayer/evm"
+	"github.com/smartcontractkit/chainlink/v2/internal/testdb"
```

Comment:
Is this going to be migrated to common?

Reply:
yes, i think we would need to mv some pgtest, testdb, dialects to common or their own module and evm code to integration repo.

Reply:
Is this helpful? https://github.com/smartcontractkit/chainlink-common/pull/450